### PR TITLE
LRQA-62049 part 3 | Remove the hard pause in add portlet macro and instead waits for portletReady event

### PR DESCRIPTION
**Pull request by john-co**
Remove the hard pause in add portlet macro and instead waits for portletReady event. This depends on commit from LRQA-65670 from https://github.com/liferay-frontend/liferay-portal/pull/884

**Test Plan**
- pass ci:test:acceptance
- pass ci:test:frontend

**JIRA Ticket**
https://issues.liferay.com/browse/LRQA-62049

### CODE REVIEW

**Comment by john-co** [comment]
Reviewed. Test failures are unrelated. :heavy_check_mark: 

analyzed in https://github.com/john-co/liferay-portal/pull/387

**Comment by john-co** [comment]
Draft until https://github.com/liferay-frontend/liferay-portal/pull/884 is merged.

**Comment by john-co** [comment]
cc/ @calvinctvo

**Review by zhangyan0701 — APPROVED** [review]

File: portal-web/test/functional/com/liferay/portalweb/macros/LiferayEvent.macro [hunk 1]
```diff
@@ -0,0 +1,10 @@
+definition {
+
+	macro waitForPortletReady {
+		WaitForLiferayEvent(
```
Comment (jbalsas) [inline portal-web/test/functional/com/liferay/portalweb/macros/LiferayEvent.macro:4]: Not necessarily to do now, but it would be great if you could assess something in the event payload other than `eventName` to make sure you don't get 2 `portletReady` events and mistakenly pick the one that is not related to the thing you're waiting for.

Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/macros/LiferayEvent.macro:4]: follow up ticket: https://issues.liferay.com/browse/LRQA-65793

**Comment by john-co** [comment]
ci:forward
